### PR TITLE
Local Stack - Metaphysics

### PR DESCRIPTION
**Pull request by icirellik**
Brings metaphysics to the local stack:

force is accessible on localhost:5000
metaphysics is accessible on localhost:3000

Please consult the instructions for setup information:

https://github.com/artsy/artsy-local/blob/eb8b090d486164a9a20cb297bdeb7ba54e133d10/README.md

### CODE REVIEW

File: env/metaphysics.env [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# Service Dependencies
+######################################################################
+
+# AWS (localstack)
+AWS_ACCESS_KEY_ID=local
+AWS_BUCKET_PATH=metaphysics/heapsnapshots/staging
```
Comment (icirellik) [inline env/metaphysics.env]: Need to double check if AWS is only here because we previously uploaded our heap dumps to a bucket.

Comment (joeyAghion) [reply env/metaphysics.env]: I think so: https://github.com/artsy/metaphysics/search?q=aws_access_key_id

File: README.md [hunk 2]
```diff
@@ -33,11 +25,31 @@ export AWS_ACCOUNT_ID=
 - `AWS_DEFAULT_REGION` - The aws region commands will execute in.
```
Comment (joeyAghion) [inline README.md:25]: Maybe we can default this?

File: README.md [hunk 3]
```diff
@@ -49,7 +61,6 @@ have a hard time allocating resources. If you ran into issues, try
```
Comment (joeyAghion) [inline README.md:61]: And eventually: `docker-compose down` to spin down the stack.

**Review by joeyAghion — APPROVED** [review]
This worked for me, at least to be able to visit a degenerate home page on `localhost:5000`.

File: README.md [hunk 4]
```diff
@@ -33,11 +25,31 @@ export AWS_ACCOUNT_ID=
...
+
+```
+aws ecr get-login-password --region ${AWS_DEFAULT_REGION} | docker login --username AWS --password-stdin ${AWS_ACCOUNT_ID}.dkr.ecr.${AWS_DEFAULT_REGION}.amazonaws.com
+```
+
+### Step Three: Run Setup
+
+Ensure that Docker is installed on your system using the proper instructions from their [website](https://docs.docker.com/get-docker/)
+
+**NOTE: If you are running on a Mac you should adjust your memory and cpu allocations as the default will be to low**
```
Comment (mc-jones) [inline README.md:38]: Might be good to include, what we think is a reasonable default.

**Review by sonecabr — APPROVED** [review]

**Pull request merged**